src/features/shared-link-settings-modal/ExpirationSection.js

```diff
@@ -29,6 +29,7 @@ const ExpirationSection = ({
     const datepicker = (
         <div>
             <DatePicker
+                dateFormat="utcTime"
```

Comment:
I'd be a little cautious since `SharedLinkSettingsModal` is used in the ContentSharing element and Canvas, probably other apps as well. there might be an expectation now that this returns a date in the local timezone so that this change could break those usages while fixing EUA.

Reply:
Ok thanks for the heads up. I will confirm in these places as well. The issue here is that the backend (monolith) for link settings expects it in this format so the current implementation is broken and showing inconsistencies in other parts of the app (folder settings)

Reply:
@tjuanitas just pushed an update to make this set by the `SharedLinkSettingsModal` component so I can set this through EUA and keep the changes isolated